src/pages/contact_us/_offices.js

```diff
@@ -1,8 +1,7 @@
 import React from 'react'
 import { graphql, useStaticQuery } from 'gatsby'
 import styled from 'styled-components'
-import { Text, LocalizedLinkText } from 'components/elements/typography'
-import { Header, QueryImage } from 'components/elements'
+import { Header, QueryImage, Text, LocalizedLinkText } from 'components/elements'
```

Comment:
Was this the online place where the import needed to be updated after the splitting of the component into separate files?

Reply:
Yup...the others were already importing LocalizedLinkText from 'components/elements'